script.js

```diff
@@ -0,0 +1,133 @@
+// 1. Implement the getGreeting function.
+
+function getGreeting(firstName, lastName) {
+    return 'Hi, "' + firstName + ' ' + lastName + '". What is up?';
+}
+
+const greeting = getGreeting('John', 'Smith');
+console.log(greeting);
+
+// 2. Implement the isThisMyName function that returns true if provided with your name.
+// Otherwise, return false.
+
+function isThisMyName(myName) {
+    return myName === 'Mikolaj'
+}
+
+console.log(isThisMyName('Mikolaj'));
+
+// 3. Implement the isThisBoolean  function that returns true  if provided with a boolean.
+
+function isThisBoolean(value) {
+    return typeof value === 'boolean';
+}
+
+console.log(isThisBoolean(true));  //true
+console.log(isThisBoolean(false));  // true
+console.log(isThisBoolean('true')); // false
+
+// 4.Implement the getCircleArea  function.
+
+function getCircleArea(radiusNumber) {
+    return Math.PI * radius * radius;
+}
+
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area); // 78.53981633974483
+
+// 4.5. Implement the getCirclesAreaSum function that uses the getCircleArea function under the hood
+
+function getCirclesAreaSum(firstRadius, secondRadius) {
+    const firstCircleArea = Math.PI * firstRadius * firstRadius;
+    const secondCircleArea = Math.PI * secondRadius * secondRadius;
+    return firstCircleArea + secondCircleArea;
+}
+
+const circlesAreaSum = getCirclesAreaSum(5, 10);
+console.log(circlesAreaSum); // 392.69908169872417
+
+// 4.75 Given a month as an integer from 1 to 12, return to which quarter of the year it belongs as an integer number.
+
+function isQuarter(monthNumber) {
+    return Math.ceil(monthNumber / 3);
+}
+
+console.log(isQuarter(12));
+
+// 5. Write the convertMinutesToHours function.
+
+function convertMinutesToHours(hoursNumber) {
+    return hoursNumber / 60;
+}
+
+const hours = convertMinutesToHours(75);
+console.log(hours); // 1.25
+
+// 6. Write the isTextUppercase  function.
+
+function isTextUppercase(text) {
+    return text === text.toUpperCase();
+}
+
+console.log(isTextUppercase('Hello')); // false
+console.log(isTextUppercase('HELLO')); // true
+
+// 7. Create a function that checks if a number n is divisible by two numbers x AND y.
+// All inputs are positive, non-zero numbers.
+
+function isDivisibleByTwoNumbers(nNumber, xNumber, yNumber) {
```

Comment:
Is this the exercise you wanted to do?
https://www.codewars.com/kata/5545f109004975ea66000086/javascript

If that's the case, your function would not pass the tests on Codewars